src/contexts/LoginContext.tsx

```diff
@@ -33,45 +34,29 @@ const useLoginContext = (): LoginContextProps => {
 const LoginProvider = ({
   children,
 }: PropsWithChildren<Record<string, never>>): JSX.Element => {
-  const [storedUserId, setStoredUserId, removeStoredUserId] = useLocalStorage(
-    LOCAL_STORAGE_KEYS.GithubId,
-    "Unknown user"
-  )
-  const [
-    storedUser,
-    setStoredUser,
-    removeStoredUser,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.User, { email: "", contactNumber: "" })
-  const [
-    storedUserEmail,
-    setStoredUserEmail,
-    removeStoredUserEmail,
-  ] = useLocalStorage(LOCAL_STORAGE_KEYS.Email, "Unknown email")
-
   const [, , removeSites] = useLocalStorage(
     LOCAL_STORAGE_KEYS.SitesIsPrivate,
     false
   )
+  const [storedUserId, setStoredUserId] = useState("")
+  const [storedUserContact, setStoredUserContact] = useState("")
+  const [storedUserEmail, setStoredUserEmail] = useState("")
   const verifyLoginAndSetLocalStorage = useCallback(async () => {
```

Comment:
does this still set local storage? else we should update the function name

Reply:
Ah yeah that's a good point, changed in https://github.com/isomerpages/isomercms-frontend/pull/1198/commits/c0295d7bbe88a08417dc0a8078120363b6b0e9d1